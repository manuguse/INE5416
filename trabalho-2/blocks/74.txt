1 1 4 8 8 13 13 17 21 21
1 4 4 9 13 13 17 17 21 21
1 1 4 4 4 14 17 17 21 21
1 5 5 4 10 14 14 14 19 19
1 5 5 10 10 10 14 19 19 22
2 2 5 11 10 15 15 19 19 22
2 2 5 11 11 16 15 15 22 22
3 3 6 6 7 15 15 20 23 23
3 3 7 7 7 7 15 18 24 24
3 3 3 12 12 7 18 18 18 25