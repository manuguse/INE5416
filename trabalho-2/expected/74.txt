7 1 6 2 1 3 2 5 6 4
6 3 5 1 4 1 3 4 5 2
4 5 4 2 7 4 2 1 3 1
3 2 6 1 5 1 5 3 6 3
2 1 5 2 3 4 2 4 5 4
4 2 4 3 1 7 5 1 2 3
3 1 3 1 2 1 4 3 1 2
6 4 1 2 5 6 2 1 2 1
5 3 6 1 3 4 1 4 1 2
2 1 7 2 1 2 3 1 2 1